documentation/Troubleshooting.md

```diff
@@ -49,10 +52,16 @@ This section provides troubleshooting information for Kubemarine and Kubernetes
 This section lists all known errors with explanations and recommendations for their fixing. If an 
 error occurs during the execution of any of these procedures, you can find it here.
 
-
 ## KME0001: Unexpected exception
 
-```
+### Description
+This error occurs when an unexpected exception is encountered during runtime and has not yet been assigned a classifying code.
+
+### Alerts
+- **Alert:** TASK FAILED - `KME001: Unexpected exception`
```

Comment:
What is the meaning of `Alerts` section? I thought it is related to monitoring alerts, which are not applicable for kubemarine CLI

Reply:
I believe the Alerts section helps to quickly identify the detected issue